data-prepper-plugins/mutate-string-processors/src/main/java/com/amazon/dataprepper/plugins/processor/mutatestring/SplitStringProcessorConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.mutatestring;
+
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+public class SplitStringProcessorConfig {
+    public static class Entry {
+
+        @NotEmpty
+        @NotNull
+        private String source;
+
+        @NotEmpty
+        @NotNull
+        private String delimiter;
+
+        public String getSource() {
+            return source;
+        }
+
+        public String getDelimiter() {
+            return delimiter;
+        }
+
+        public Entry(final String source, final String delimiter) {
+            this.source = source;
+            this.delimiter = delimiter;
+        }
+    }
```

Comment:
There needs to be a `List<Entry> entries` for this configuration class in order for the yaml to work correctly.

Reply:
Is this in reference to the Logstash Converter?

Reply:
No, the config file needs to have that list in order to have the user enter in their entries in the yaml file